static/js/containers/ChannelPage.js

```diff
@@ -62,23 +104,26 @@ class ChannelPage extends React.Component {
   }
 
   render() {
-    const { channels, postsForChannel, posts } = this.props
+    const { channels, postsForChannel, posts, subscribedChannels } = this.props
     const channelName = getChannelName(this.props)
 
     return (
       <Loading
         restStates={[channels, postsForChannel]}
-        renderContents={() => this.renderContents(channelName, channels, postsForChannel, posts)}
+        renderContents={() => this.renderContents(
+          channelName, channels, postsForChannel, posts, subscribedChannels
```

Comment:
We should do away with the bulky parameter passing here and use `this` in renderContents. Then we would just define the prop as `renderContents=this.renderContents` or `renderContents=this.renderContents.bind(this)`. I made a similar comment on Nathan's PR here: https://github.com/mitodl/open-discussions/pull/93#discussion_r131759798